### DISCUSSION TITLE
Chrome dino?

### DISCUSSION THREAD

**Discussion by Flyingbacen** [minbrowser/min#2402] [Q&A]
Went to chrome://dino because I was curious what it went to since I knew it used chromium. Gave me a no internet page, which makes me curious is chrome://dino just calls some internet disconnected page, with an additional feature to zoom in?

**Comment by PalmerAL** [comment]
Electron uses the Chromium rendering engine, but not the UI parts (which includes the dino game), so we implement our own page: https://github.com/minbrowser/min/blob/master/pages/error/error.js

**Comment by Tedd-Codes** [comment]
I think this would be fun
